src/main/java/io/github/battlepass/logger/containers/LogContainer.java

```diff
@@ -13,6 +14,10 @@ public static LogContainer of(String message, Player player) {
         return new BasicPlayerContainer(message, player);
     }
 
+    public static LogContainer of(QuestExecution questExecution) {
+        return new QuestExecutionContainer(questExecution);
+    }
```

Comment:
This convention should be followed throughout the plugin.

Reply:
As in to change this part or change other existing parts of the plugin to match this format?

Reply:
As in any method that constructs a class should be above the class' constructor. Or below, just consistent across the plugin.

Reply:
Moved down under the constructor.